app/models/attachment.rb

```diff
@@ -29,6 +29,8 @@ class Attachment < ApplicationRecord
     allow_blank: true, greater_than: 0
   }
 
+  default_scope { includes(:attachment_data) }
```

Comment:
What's your guess for how many places in the code would require changing if this wasn't done through a default scope?

I guess this makes sense if it's always useful, but is that definitely the case?

Reply:
Hi @cbaines , thanks for this! We reviewed this again and found we can avoid using `default_scope`

Reply:
Awesome :+1: